workspace/solution.go

```diff
@@ -88,3 +88,13 @@ func (s *Solution) PathToParent() string {
 	}
 	return filepath.Join(dir, s.Track)
 }
+
+func createIgnoreSubdir(path string) error {
+	path = filepath.Join(path, ignoreSubdir)
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		if err := os.Mkdir(path, os.FileMode(0755)); err != nil {
```

Comment:
If you use `MkdirAll` instead of `Mkdir` you don't have to check that it doesn't exist first.